packages/terra-popup/src/Popup.jsx

```diff
@@ -0,0 +1,368 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Portal from 'react-portal';
+import PopupContent from './_PopupContent';
+import PopupArrow from './_PopupArrow';
+import PopupOverlay from './_PopupOverlay';
+import TetherComponent from './_TetherComponent';
+import './Popup.scss';
+
+const MIRROR_LR = {
+  center: 'center',
+  left: 'right',
+  right: 'left',
+};
+
+const MIRROR_TB = {
+  middle: 'middle',
+  top: 'bottom',
+  bottom: 'top',
+};
+
+const BASE_WIDTH = 16;
+const BASE_HEIGHT = 9;
```

Comment:
Would we want to export these values for consumers who want to may want to calculate what popup size to use based on the content?